tests/test_api_v10.py

```diff
@@ -38,6 +40,40 @@ def mocked_resultsdb():
         yield mocked_resultsdb
 
 
+class TestAuthorizationWarning:
+    @patch('waiverdb.api_v1._verify_authorization')
+    def test_user_authorized(self, mock_verify_auth, mocked_user, app):
```

Comment:
Where is `mock_verify_auth` defined?

Reply:
`@patch('waiverdb.api_v1._verify_authorization')`

Reply:
Ah, I see!